docs/developer.md

```diff
@@ -0,0 +1,203 @@
+# Developer Docs
+
+In order to contribute to this project, make sure you have go 1.18 installed
+on your local machine.
+
+## Dependencies
+
+The build system assumes the following binaries are in the `PATH`:
+
+```
+make git go npm kind podman (or docker)
+```
+Once these tools are installed, run `make tools` to install all required
+project dependencies to ``tmp/bin``
+```
+make tools
+```
+## Development Environment Setup
+
+To setup your development, it is recommended to run the helper script
+`hack/kind/setup.sh`. This script does the following
+    * sets up a Kind cluster
+    * installs OLM
+    * sets up a local registry to push locally built operator and bundle images
+
+```sh
+./hack/kind/setup.sh
+```
+
+Once done, the cluster can be deleted by running
+
+```
+kind delete cluster --name obs-operator
+```
+
+## Running End to End tests
+
+Running E2E locally against the kind cluster that was setup following the
+instructions above is as easy as running `./test/run-e2e.sh`.
+
+```sh
+ ./test/run-e2e.sh
+```
+**NOTE:** `./test/run-e2e.sh --help` shows options that are useful when
+rerunning tests.
+
+
+# Running Operator In Kind Cluster
+
+
+Run `./hack/kind/setup.sh` to create a Kind cluster named `obs-operator` and
```

Comment:
Isn't this info a repeat of what we have in `Development Environment Setup`?

Reply:
yes, but this section is referred by the README.md, so I felt repeating gives a better experience when you jump from readme to this section. 

Perhaps we should add a `(see above)`

Reply:
Ahhh I see, sorry I hadn't noticed this! I agree it provides a better experience if you are coming from README.md, but if you just go to developer.md directly you will be faced with duplicate information. 🤔 In my opinion, either we:

- do it as you suggested or 
- we change the redirect to `Development Environment Setup`

Do as you please both work for me :)